x-pack/plugins/ml/public/data_frame/pages/access_denied/directive.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+// @ts-ignore
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+
+const module = uiModules.get('apps/ml', ['react']);
+
+import { I18nContext } from 'ui/i18n';
+import { InjectorService } from '../../../../common/types/angular';
+
+import { Page } from './page';
+
+module.directive('mlDataFrameAccessDenied', ($injector: InjectorService) => {
+  return {
+    scope: {},
+    restrict: 'E',
+    link: (scope: ng.IScope, element: ng.IAugmentedJQuery) => {
+      const kbnBaseUrl = $injector.get<string>('kbnBaseUrl');
+      const kbnUrl = $injector.get<any>('kbnUrl');
+
+      const goToKibana = () => {
+        window.location.href = uiChrome.getBasePath() + kbnBaseUrl;
```

Comment:
Any reason this is not using a template literal to create the href?

Reply:
Copy pasted this from the general access denied page for anomaly detection :)